OpenSim/Simulation/Model/Model.h

```diff
@@ -220,18 +220,23 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Model, ModelComponent);
     //--------------------------------------------------------------------------
 public:
 
-    /** Default constructor creates a %Model containing only the ground Body
+    /** Default constructor creates a %Model containing only the Ground frame
     and a set of default properties. */
     Model();
 
     /** Constructor from an OpenSim XML model file. 
+    NOTE: The Model is read in (deserialized) from the model file, which means
+    the properties of the Model and its components are filled in from values in
+    the file. In order to evaluate the validity of the properties (e.g. Inertia
+    tensors, availability of Mesh files, and to traverse the tree of the
+    Model's subcomponents) one must invoke Model::finalizeFromProperties() 
+    first. Model::initSystem() invokes finalizeFromProperties() on its way to
+    creating the System and initializing the State.
```

Comment:
> Maybe another term would be better?

_Catalog_/_Subscribe_ a Component with a Model? Update a Model's _inventory_ of Components?